config/fam_memoryserver_config.yaml

```diff
@@ -13,7 +13,8 @@
 # where interface address is hostname (fqdn)/ip4 address where grpc port should be exposed.
 # portno is port to be used by OpenFAM for communication between Memory Server and
 # Client Interface Service.
-rpc_interface: 127.0.0.1:8789
+# This field is reserved for future use and hence commented for now.
```

Comment:
May be something like this is better :
This field is reserved for future use. Please use the command line options to pass the RPC interface, while starting up the memory_server.